inst/rmarkdown/templates/arxiv/skeleton/skeleton.Rmd

```diff
@@ -20,7 +20,9 @@ keywords:
   - bloo
   - these are optional and can be removed
 bibliography: references.bib
-output: rticles::arxiv_article
+output: 
+  rticles::arxiv_article:
+    keep_tex: true
```

Comment:
@Athanasiamo just a precision about this change I kept: 

`keep_tex = TRUE` is already the default in `arxiv_article()`. So even if you don't mention it in the YAML header, the tex file will be kept. 

FYI